docs/generate_server.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/spec/openapi.yaml -g python-aiohttp -o /local/server_new --git-repo-id athenianco/athenian-api --package-name athenian.api
+docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/docs/openapi.yaml -g python-aiohttp -o /local/server_new --git-repo-id athenianco/athenian-api --package-name athenian.api
```

Comment:
Right, this was from the times when `/spec/openapi.yaml` was a symbolic link to `/server/athenian/api/openapi/openapi.yaml`